### DISCUSSION TITLE
First note will pause slightly on some beatmap

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#30948] [Q&A] [answered]
use 0.25x to play the video

I noticed this during my friend's play.

https://github.com/user-attachments/assets/e6d4d7a5-6a15-48e8-bdd5-2c2c35a52b5f

Focus on the first note, it will pause slightly when it reaches its strike time.

beatmap link: https://osu.ppy.sh/beatmapsets/1027900#mania/2149349

and there shouldn't be any timing point that should cause a pause.

I don't know if it's unique to mania, and this problem also appear between stable and lazer

his dxdiag file
[DxDiag(1).txt](https://github.com/user-attachments/files/17977473/DxDiag.1.txt)

Then I tested it on my machine and it also happened

https://github.com/user-attachments/assets/de880123-0a42-4f47-98da-041311a55255

my dxdiag file and log
[DxDiag.txt](https://github.com/user-attachments/files/17977537/DxDiag.txt)
[compressed-logs.zip](https://github.com/user-attachments/files/17977614/compressed-logs.zip)

But my video looks more like the entire playfield slows down instantly after hitting it and then recovers.

By the way, I think this problem may have been reported before, but I didn't find any related issues through the keywords first note start stuck. I apologize if it has been reported before.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/22488

known issue with bass
